bin/cluster

```diff
@@ -294,7 +294,7 @@ if __name__ == '__main__':
         meta_parser.add_argument('cluster_id', help='prefix for cluster VM names')
 
     meta_parser.add_argument('-t', '--deployment-type',
-                             choices=['origin', 'online', 'enterprise'],
+                             choices=['origin', 'online', 'enterprise', 'openshift-enterprise'],
```

Comment:
We should probably also add 'atomic-enterprise' to the list of deployment-types and the product-type flag below should probably be removed, since it's been removed in the ansible playbooks.